### PR TITLE
Simple LunarVim docker image

### PR DESCRIPTION
**Pull request by colutti**
This is a simple LunarVim docker image based on the python official image. I am installing all Lunar dependencies as well as neovim.

### CODE REVIEW

**Comment by abzcoding** [comment]
we can possibly use [Github Container registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry) and generate new containers using our CI or sth.

and people can just do
```bash
docker pull ghcr.io/ChristianChiarulli/LunarVim:0.5.1
# or without version to get the latest
```

File: utils/docker/script.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+npm install tree-sitter-cli neovim
+
+# Installing Neovim
+mkdir -p /tmp/neovim
+cd /tmp/neovim || exit
+curl -L -o nvim.appimage https://github.com/neovim/neovim/releases/download/nightly/nvim.appimage
+chmod u+x nvim.appimage
+./nvim.appimage --appimage-extract
+mv squashfs-root /usr/local/neovim
+ln -s /usr/local/neovim/usr/bin/nvim /usr/bin/nvim
```
Comment (abzcoding) [inline utils/docker/script.sh:19]: can we remove `/tmp/nvim.appimage` afterwards :pleading_face:

Comment (colutti) [reply utils/docker/script.sh:19]: done

**Review by abzcoding — APPROVED** [review]
LGTM :rocket:

**Pull request merged**